<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KdPc3dShowClass</class>
 <widget class="QMainWindow" name="KdPc3dShowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KdPc3dShow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_openFile">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reShow">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重新显示</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>111</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Step</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_xStep">
        <property name="text">
         <string>X Step：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_xStep">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_yStep">
        <property name="text">
         <string>Y Step：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_yStep">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_zStep">
        <property name="text">
         <string>Z Step：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_zStep">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Dimension">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>490</y>
      <width>81</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Dimension</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>81</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_row">
        <property name="text">
         <string>Row：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_row"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_col">
        <property name="text">
         <string>Col：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_col"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Coordinate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>91</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Coordinate</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_xCod">
        <property name="text">
         <string> X：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_xCod">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_yCod">
        <property name="text">
         <string> Y：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_yCod">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_zCod">
        <property name="text">
         <string> Z：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_zCod">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_saveFile">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>490</y>
      <width>151</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotation</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_RotationAxis">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_RotationAngle">
        <property name="text">
         <string>angle</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_xRotation">
        <property name="text">
         <string> X：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_xRotationAxis">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_yRotation">
        <property name="text">
         <string> Y：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_yRotationAxis">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_RotationAngle">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_zRotation">
        <property name="text">
         <string> Z：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_zRotationAxis">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>231</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Roi</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>224</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_xRoiRange">
        <property name="text">
         <string>X range            </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_xRoiStart">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_xLine">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_xRoiEnd">
        <property name="text">
         <string>60</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_yRoiRange">
        <property name="text">
         <string>Y range            </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_yRoiStart">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_yLine">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_yRoiEnd">
        <property name="text">
         <string>15</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_zRoiRange">
        <property name="text">
         <string>Z range            </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_zRoiStart">
        <property name="text">
         <string>-3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_zLine">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_zRoiEnd">
        <property name="text">
         <string>15</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_showDepthColor">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>深度显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KdPc3dShow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_openFile</sender>
   <signal>clicked()</signal>
   <receiver>KdPc3dShowClass</receiver>
   <slot>on_openFile_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reShow</sender>
   <signal>clicked()</signal>
   <receiver>KdPc3dShowClass</receiver>
   <slot>on_reShow_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>767</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_saveFile</sender>
   <signal>clicked()</signal>
   <receiver>KdPc3dShowClass</receiver>
   <slot>on_saveFile_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_showDepthColor</sender>
   <signal>clicked()</signal>
   <receiver>KdPc3dShowClass</receiver>
   <slot>on_showDepthColor_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>591</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_openFile_clicked()</slot>
  <slot>on_reShow_clicked()</slot>
  <slot>on_saveFile_clicked()</slot>
  <slot>on_showDepthColor_clicked()</slot>
 </slots>
</ui>
